docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,210 @@
+::::
+
+::::{dropdown} 6. Will the software that my group uses exclusively on our server(s) be available on other servers?
+
+Yes, all software used in the private partitions is available on all servers. Software that is restricted to a particular group’s license will remain restricted to members of that group, but they will be able to use that software on any server. 
+::::
+
+::::{dropdown} 7. What hardware will be part of the `lowpriority` partition?
+All PI-owned hardware purchased on or after 2019 will be part of the `lowpriority` partition.
+::::
+
+::::{dropdown} 8. Do you have a clear inventory of the various resources available through the `lowpriority` partition?
+
+Throughout the testing phase, RC will be updating its `low priority` documentation (see {ref}`low-priority`)to reflect the resources that are part of public and private partitions. Please check the documentation (see {ref}`low-priority`) regularly for latest information. See our CPUs ({ref}`hardware-overview`) and GPUs ({ref}`working-gpus`) pages.
+::::
+
+:::{button-link} faq.html
+:color: primary
+:align: center
+Back to the top
+:::
+(low-priority-preemption)=
+## Preemption
+
+::::{dropdown} 1. What does 'preemption' or 'preemptable job' mean?
+In SLURM terminology, preemption is a scheduling mechanism that involves stopping/suspending one or more low priority jobs, to accommodate a high priority job (i.e. job submitted by the partition owner’s group member). A low priority job that can get suspended in this manner is also known as a ‘preemptable’ job. Preemptable jobs can run on private hardware owned by other research groups, with the potential risk of being suspended before their wall time ends if the private partition’s group member submits a job and requests for resources.
+::::
+
+::::{dropdown} 2. What do I need to know about preemption and are there any drawbacks for my research group?
+
+**For private partition owners:** 
+When low priority jobs are running on your hardware, you may experience a slight delay (~30s) when submitting your first job to your private partition. This is the time it will take the scheduler to kill/preempt low priority jobs running on your partition. This new service is designed to minimize that delay.
+
+**For users of the** `lowpriority` **partition:** 
+Submitting jobs to this partition will make your jobs preemptable. That means they may be stopped/suspended and re-queued at any time. We recommend that you use strategies such as checkpointing (see {ref}`checkpoint-jobs`) to take full advantage of the `lowpriority` partition. RC is working on detailed documentation (see {ref}`low-priority`) and training sessions that will be available later in the Fall 2023 semester, once the service is ready to be released.
+::::
+
+::::{dropdown} 3. How does re-queuing of preempted (terminated) low priority jobs work?
```

Comment:
Maybe change this to something like, What happens if my job is preempted?

Reply:
Would 'What happens if my job is preempted on low-priority?' work? Maybe being explicit on which partition might head off further questions?